azure-cosmos-db/1.0/modules/ROOT/pages/azure-cosmos-db-connector-examples.adoc

```diff
@@ -0,0 +1,1517 @@
+        xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core" xmlns:http="http://www.mulesoft.org/schema/mule/http"
+        xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+http://www.mulesoft.org/schema/mule/azure-cosmos-db-connector-mule-4 http://www.mulesoft.org/schema/mule/azure-cosmos-db-connector-mule-4/current/mule-azure-cosmos-db-connector-mule-4.xsd">
+
+    <configuration-properties file="mule-app.properties"/>
+
+    <http:listener-config name="HTTP_Listener_config">
+        <http:listener-connection host="0.0.0.0" port="8081"/>
+    </http:listener-config>
+    <azure-cosmos-db-connector-mule-4:key-token-config name="Cosmos_Db_Connector_Config">
+        <azure-cosmos-db-connector-mule-4:key-token-auth-connection key="${config.masterKey}"
+                                                                       version="${config.version}"
+                                                                       baseUri="${config.baseUri}"/>
+    </azure-cosmos-db-connector-mule-4:key-token-config>
+    <azure-cosmos-db-connector-mule-4:resource-token-config name="Cosmos_Db_Connector_Config_Resource_Token">
+        <azure-cosmos-db-connector-mule-4:resource-token-auth-connection resourceToken="#[vars.resourceToken]"
+                                                                         version="${config.version}"
+                                                                         baseUri="${config.baseUri}"/>
+    </azure-cosmos-db-connector-mule-4:resource-token-config>
+    <flow name="delete-stored-procedure-flow">
+           <http:listener config-ref="HTTP_Listener_config" path="/delete-stored-procedure"/>
+           <azure-cosmos-db-connector-mule-4:delete-stored-procedure config-ref="Cosmos_Db_Connector_Config"
+                                                                     dbIdUriParam="${config.database}"
+                                                                     collIdUriParam="${config.collection}"
+                                                                     sprocIdUriParam="Mule_CosmosDB_DemoApp_StoredProcedureId">
+           </azure-cosmos-db-connector-mule-4:delete-stored-procedure>
+           <logger level="INFO" message="#[payload]"/>
+       </flow>
+</mule>
+----
+
+=== Steps for Running This Example
+
+. Verify that your connector is configured.
+. Save the project.
+. From a web browser, test the application by entering `http://localhost:8081/delete-stored-procedure`.
+
+
+[[create-permission]]
+== Create Permission
+
+This Mule creates a permission to access a resource for a user.
```

Comment:
This Mule flow creates a user permission for a specific resource???
This Mule flow creates a user permission???